src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DoubleInput.cs

```diff
@@ -28,7 +28,7 @@ public void CustomizeView(DoubleInput nodeModel, NodeView nodeView)
 
             tb.DataContext = nodeModel;
 
-            tb.BindToProperty(new Binding("Value")
+            tb.BindToProperty(new Binding("NumericalValue")
             {
                 Mode = BindingMode.TwoWay,
                 Converter = new DoubleInputDisplay(),
```

Comment:
@ikeough I like the idea of putting the validation in the view layer on the converter. Will give that a shot.

But I'm not sure `NumericalValue` can be a double type since the issue is that we want to continue supporting range expressions for old graphs, which means the node's value might really be a string.

An alternative is to remove this functionality from number nodes, and implement some conditional migration to code blocks (for range expressions), but I don't think that will solve the issue for graphs that already use numbers in flood.

The `convert` ,`can convert` methods are existing public methods.